database/migrations/sqlite/1728374074113685_release_environments.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS releases_new
+(
+    eslVersion INTEGER, -- internal ID for ESL
+    releaseVersion INTEGER, -- release ID given by the client that calls CreateApplicationVersion
+    created TIMESTAMP,
+    appName VARCHAR,
+    manifests VARCHAR, -- json blob, map where each key is an environment and each value is a manifest
+    metadata VARCHAR,  -- json blob about the release, sourceAuthor, sourceCommitId, etc
+    deleted  BOOLEAN,
+    environments VARCHAR,
```

Comment:
environments should be an array
```suggestion
    environments VARCHAR[],
```